misocho/questioner
# #163736344 View Meetups

#### What does this PR do?
* Displays posted meetups on the homepage
#### Description of Task to be completed?
* Display posted meetups on the homepage
#### How should this be manually tested?
* Clone the repo
* Navigate to the project directory
* Open index.hml in the templates folder with your browser
* You should be able to view posted meetups
#### What are the relevant pivotal tracker stories?
* https://www.pivotaltracker.com/n/projects/2235306/stories/163736344
#### Screenshots (if appropriate)
![screenshot_2019-02-19_18-41-19](https://user-images.githubusercontent.com/29132159/53027448-f94e7f80-3475-11e9-8336-54e2833cba48.png)
